.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+jobs:
+  build:
+    working_directory: /home/circleci/.go_workspace/src/github.com/grafana/metrictank
+    machine: true
+    steps:
+      - run: go version
+      - run: echo $GOPATH
+      - run: docker version
+      - run: docker-compose version
+      - checkout
+      - run: scripts/depends.sh
+      - run: scripts/qa/vendor.sh
+      - run: scripts/build.sh
+      - run: scripts/build_tools.sh
+      - run: scripts/build_docker.sh
+      - run: go test -v -race $(go list ./... | grep -v github.com/grafana/metrictank/chaos)
+      - run: go vet ./...
+      - run: scripts/qa/gofmt.sh
+      - run: scripts/qa/ineffassign.sh
+      - run: scripts/qa/go-generate.sh
+      - run: scripts/qa/end2end.sh
+      - run: scripts/qa/misspell.sh
+      - run: scripts/qa/unused.sh
+      - run: scripts/qa/tools_gitignore.sh
+      - run: scripts/qa/docs.sh
+      - run: scripts/build_packages.sh
+
+general:
+  artifacts:
+    - build
+
+deployment:
```

Comment:
`deployment` does not appear to be a supported top level key.   Looks like you need to use workflows